Activities/Scripting/UiPath.Scripting.Activities.Design/Themes/Icons.xaml

```diff
@@ -0,0 +1,5 @@
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+
```

Comment:
leave a blank comment if this is kept for future icons, else remove it. it makes no sense to have it blank